packages/components/secrets/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@amazon-codecatalyst/blueprint-component.secrets",
+  "description": "This is a representation of a codecatalyst secret.",
+  "scripts": {
+    "build": "npx projen build",
+    "bump": "npx projen bump",
+    "clobber": "npx projen clobber",
+    "compile": "npx projen compile",
+    "default": "npx projen default",
+    "eject": "npx projen eject",
+    "eslint": "npx projen eslint",
+    "package": "npx projen package",
+    "post-compile": "npx projen post-compile",
+    "post-upgrade": "npx projen post-upgrade",
+    "pre-compile": "npx projen pre-compile",
+    "test": "npx projen test",
+    "unbump": "npx projen unbump",
+    "upgrade": "npx projen upgrade",
+    "watch": "npx projen watch",
+    "projen": "npx projen --no-post",
+    "component:package": "yarn build && yarn package",
+    "npm:push": "yarn bump && yarn component:package && yarn npm:publish",
+    "npm:publish": "npm publish dist/js/*.tgz"
+  },
+  "devDependencies": {
+    "@amazon-codecatalyst/blueprint-util.projen-blueprint-component": "*",
+    "@amazon-codecatalyst/blueprints.blueprint": "*",
+    "@types/node": "^18",
+    "@typescript-eslint/eslint-plugin": "^5",
+    "@typescript-eslint/parser": "^5",
+    "eslint": "^8",
+    "eslint-import-resolver-node": "*",
+    "eslint-import-resolver-typescript": "*",
+    "eslint-plugin-import": "*",
+    "npm-check-updates": "^16",
+    "projen": "*",
+    "standard-version": "^9",
+    "ts-node": "^10",
+    "typescript": "^5.3.3"
+  },
+  "peerDependencies": {
```

Comment:
I dont think you need this explictly listed as a peer. 
```
@amazon-codecatalyst/blueprints.blueprint"
```

Reply:
fixed